controllers/apply/form-router-next/summary.js

```diff
@@ -76,6 +76,18 @@ module.exports = function(formBuilder) {
             }
         }
 
+        if (form.progress.isComplete) {
+            const msg = ['Apply: AFA: Summary: User shown form complete'];
+            if (res.locals.hotJarTagList) {
```

Comment:
I think `concat` will handle this `if` logic for you.

Reply:
You sure? Playing about with it, it'll add `undefined` into the array if it doesn't exist?

Reply:
Hmm. I think I was misremembering, ignore me!